0107-preventing-gaming-within-the-mobile-network.md

```diff
@@ -64,7 +64,7 @@ If it is identified that Service Providers are abusing their power to turn off M
 Deployers who have an outdoor Wi-Fi Access Point asserted higher than 100 feet will need to update the assertion height to a maximum of 100 feet.
 
 ## Rationale and Alternatives
-An alternative would be to do nothing and allow gaming to continue.
+One alternative is to give the MOBILE subDAO the power to ban hotspots, similar to how CrowdSpot operated on the IOT network. However, as there were flaws to that method, CrowdSpot no longer exists. 
```

Comment:
```suggestion
One alternative is to give the MOBILE network the power to ban Hotspots, similar to how Crowdspot operated on the IOT network. However, as there were flaws to that method, Crowdspot no longer exists due to the overhead required to operate that particular implementation.
```

Reply:
suggestions on language.

Reply:
> suggestions on language.

Thanks. accepted suggestion. This HIP is now ready for voting whenever you merge it.